### PR TITLE
Verify rhods_aggregated_availability Detects Downtime In Jupyterhub And Dashboard

### PR DESCRIPTION
**Pull request by pablofelix**
Signed-off-by: Pablo Felix Estevez Pico <pestevez@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/ODH/Prometheus/Prometheus.robot [hunk 1]
```diff
@@ -214,6 +213,17 @@ Get Target Endpoints Which Have State Up
     ${links}=    Replace String    ${links}    "    ${EMPTY}
     @{links}=    Split String  ${links}  \n
     [Return]    ${links}
+    
+Get Availability Value Change Date 
+    [Documentation]    Returns date of last availability value change
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/Prometheus/Prometheus.robot:218]: hmm it looks that it is not returning the date of the last value change, but the date of the last measurement. Is that correct?

The change detection seems to be implemented by these other lines at tests/Tests/900__high_availability/900__openshift/900__openshift.robot
```
49. ${component_downtime_start_date}=    
50.    ...    Wait Until Keyword Succeeds    1m    1s    Get Availability Value Change Date    0
```

File: tests/Resources/Page/ODH/Prometheus/Prometheus.robot [hunk 2]
```diff
@@ -225,3 +235,55 @@ Verify ODS Availability
     FOR    ${value}    IN    @{values}
         Should Not Be Equal As Strings    ${value}[1]    0
     END
+
+Verify ODS Availability Range
+    [Documentation]    Verifies that there is no downtime in ODS
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/Prometheus/Prometheus.robot:240]: same documentation as `Verify ODS Availability`

Comment (pablofelix) [reply tests/Resources/Page/ODH/Prometheus/Prometheus.robot]: Done

File: tests/Resources/Page/ODH/Prometheus/Prometheus.robot (hunk 1 above)
Comment (pablofelix) [reply tests/Resources/Page/ODH/Prometheus/Prometheus.robot]: I've modified the keyword name to clarify its purpose

**Review by bdattoma — APPROVED** [review]

**Review by mwaykole — APPROVED** [review]

**Pull request merged**